src/lib/storage/vector_compression/fixed_width_integer/fixed_width_integer_utils.hpp

```
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "storage/vector_compression/compressed_vector_type.hpp"
+#include "utils/assert.hpp"
+
+namespace opossum {
+
+inline bool is_fixed_width_integer(CompressedVectorType type) {
+  switch (type) {
+    case CompressedVectorType::FixedWidthInteger4Byte:
+    case CompressedVectorType::FixedWidthInteger2Byte:
+    case CompressedVectorType::FixedWidthInteger1Byte:
+      return true;
+    case CompressedVectorType::BitPacking:
+      return false;
+  }
+  return false;
+}
+
+inline size_t byte_width_for_fixed_width_integer_type(CompressedVectorType type) {
+  DebugAssert(is_fixed_width_integer(type), "Type must be one of the Fixed-width integer types.");
+
+  switch (type) {
+    case CompressedVectorType::FixedWidthInteger4Byte:
+      return 4u;
+    case CompressedVectorType::FixedWidthInteger2Byte:
+      return 2u;
+    case CompressedVectorType::FixedWidthInteger1Byte:
+      return 1u;
+    case CompressedVectorType::BitPacking:
+      return 0u;
+  }
+  return 0u;
```

Fail loudly (`Fail(<message>)`